src/System.Text.Encoding.CodePages/ref/System.Text.Encoding.CodePages.cs

```diff
@@ -14,4 +14,21 @@ public sealed partial class CodePagesEncodingProvider : System.Text.EncodingProv
         public override System.Text.Encoding GetEncoding(int codepage) { throw null; }
         public override System.Text.Encoding GetEncoding(string name) { throw null; }
     }
+
+    public class TestEncoderFallback : System.Text.EncoderFallback
```

Comment:
Is this a mistake? Nothing shuld be added to the ref

Reply:
Removed.